helm/prometheus-rules/templates/alerting-rules/inhibit.all.rules.yml

```
@@ -21,7 +21,7 @@ spec:
         team: phoenix
         topic: monitoring
     - alert: InhibitionKubeletDown
-      expr: up{app="kubelet"} == 0
+      expr: label_replace(up{app="kubelet"}, "ip", "$1", "instance", "(.+):\\d+") == 0
```

Turns out `up{app="cadvisor"}` has the `node` value, and the same `instance` value.
Wouldn't it be nice to have all the heavy lifting in the inhibition, so other rules remain simpler to maintain?

I think this would do the job:
 ```suggestion
            label_replace(up{app="kubelet"} * on(instance) group_left(node) up{app="cadvisor"}, "ip", "$1", "instance", "(.+):\\d+")
```

---

This means the inhibition would be on cluster_id, node instead of cluster_id, ip right?

---

oh, yes, my bad, we can't easily match on either one or the other. forget about my comment :(

---

We really need to discuss inhibitions. I added an issue draft on the board but let's discuss tomorrow